crates/forge/bin/cmd/inspect.rs

```diff
@@ -532,15 +536,19 @@ fn print_json_str(obj: &impl serde::Serialize, key: Option<&str>) -> Result<()>
     Ok(())
 }
 
-fn print_yul(yul: Option<&str>) -> Result<()> {
+fn print_yul(yul: Option<&str>, strip_comments: bool) -> Result<()> {
     let Some(yul) = yul else {
         eyre::bail!("Could not get IR output");
     };
 
     static YUL_COMMENTS: LazyLock<Regex> =
         LazyLock::new(|| Regex::new(r"(///.*\n\s*)|(\s*/\*\*.*\*/)").unwrap());
```

Comment:
This should also fix the regex

```suggestion
        LazyLock::new(|| Regex::new(r"(///.*\n\s*)|(\s*/\*\*.*?\*/)").unwrap());
```